test/scalability/README.md

```diff
@@ -0,0 +1,89 @@
+# Introduction
+
+The scalability test framework attempts to determine how the product scales in terms of concurrent SQL connections
+
+# Running
+
+
+1. Collect the data:
+
+```
+cd test/scalability
+./mzcompose run default
+```
+
+2. Start Jupyter lab
+
+```
+./mzcompose run lab
+```
+
+This is going to display a URL that you can open in your browser
+
+3. Open `scalability.ipynb`
+
+4. From the Run menu, select `Run All Cells`
+
+5. Select the workload you want charted from the drop-down
+
+6. Use drag+drop to preserve any charts
+
+## Selecting a target
+
+The framework can be directed to execute its queries against various targets:
+
+# Against your current HEAD in a container
+
+```
+./mzcompose run default --target HEAD ...
+```
+
+# Against a specific DockerHub tag:
+
+```
+./mzcompose run default --target v1.2.4 ...
+```
+
+In both of those cases, Materialize, CRDB and Python will run within the same machine, possibly interfering with each other
+
+# Against a local containerized Postgres instance
+
+
+```
+./mzcompose run default --target postgres ...
+```
+
+# Against a remote Materialize instance
+
+```
+./mzcompose run default --target=remote \--materialize-url="postgres://user:password@host:6875/materialize?sslmode=require" --cluster= ...
+```
+
```

Comment:
Optional: Add a cool screenshot somewhere? 😀

Reply:
Yes, I hope to do a demo once I am confident enough in the numbers.